spec/system/reviewables_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+describe "Reviewables", type: :system, js: true do
+  let(:review_page) { PageObjects::Pages::Review.new }
+  fab!(:admin) { Fabricate(:admin) }
+  fab!(:theme) { Fabricate(:theme) }
+  fab!(:long_post) { Fabricate(:post_with_very_long_raw_content) }
+  #
+
+  before { sign_in(admin) }
+
+  describe "when there is a reviewable with a long post" do
+    fab!(:long_reviewable) { Fabricate(:reviewable_flagged_post, target: long_post) }
```

Comment:
`target: long_post`

@jjaffeux Could this cause flakiness? Is there a better solution here?